events.js

```diff
@@ -75,10 +75,10 @@ module.exports = function (version) {
 
   function eachFrom(keys, key, iter) {
     var i = keys.indexOf(key)
-    if(!~i) return
+    if (!~i) return
```

Comment:
Request this is changed tū be more accessible

Reply:
Thanks, that is a good suggestion. I'll do that, but I'd rather do that in a separate PR together with this [line](https://github.com/ssbc/epidemic-broadcast-trees/blob/bfcd2cab26ecc2a80875dfdcb75fb2e200ce6175/events.js#L200). I'm trying to keep this PR to only trivial changes that are super easy to review.